Commons/bungee/src/cl/bgmp/bungee/CommonsBungee.java

```diff
@@ -0,0 +1,94 @@
+package cl.bgmp.bungee;
+
+import cl.bgmp.bungee.commands.HelpOPCommand;
+import cl.bgmp.bungee.commands.ServerCommands;
+import cl.bgmp.bungee.commands.privatemessage.PrivateMessageCommands;
+import cl.bgmp.bungee.commands.privatemessage.PrivateMessagesManager;
+import cl.bgmp.bungee.commands.staffchat.StaffChatCommands;
+import cl.bgmp.bungee.commands.staffchat.StaffChatManager;
+import cl.bgmp.bungee.listeners.PlayerEvents;
+import com.sk89q.bungee.util.BungeeCommandsManager;
+import com.sk89q.bungee.util.CommandExecutor;
+import com.sk89q.bungee.util.CommandRegistration;
+import com.sk89q.minecraft.util.commands.CommandException;
+import com.sk89q.minecraft.util.commands.CommandPermissionsException;
+import com.sk89q.minecraft.util.commands.CommandUsageException;
+import com.sk89q.minecraft.util.commands.MissingNestedCommandException;
+import com.sk89q.minecraft.util.commands.WrappedCommandException;
+import java.util.HashMap;
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.plugin.Listener;
+import net.md_5.bungee.api.plugin.Plugin;
+
+public class CommonsBungee extends Plugin implements CommandExecutor<CommandSender> {
+  private static CommonsBungee commonsBungee;
+  private static HashMap<String, String> privateMessagesReplyRelations;
```

Comment:
Why can't this be in `PrivateMessageManager`?

Reply:
Moved. https://github.com/ElMedievo-UdeC/Network/pull/31/commits/9f0d09da8d8f679861a8994ca7961cc8b9f69619 .